pages/knowledge/colours-and-contrast/text/README.md

```diff
@@ -0,0 +1,79 @@
+---
+navigation_title: "Colour contrast for text"
+position: 3
+changed: "2010-04-21"
+---
+
+# Colour contrast for text
+
+**There is a general minimal colour contrast level for all kinds of text. As an exception, large text can have slightly lower contrast. This requirement applies to both “real” text and rasterised text on images.**
+
+## Minimal contrast ratio
+
+The [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#contrast-minimum) of `4.5:1` against adjacent colour(s) for text in general. An exception is made for large-scale text: It is considered easier to read and can therefore be set in a lower contrast ratio of at least `3:1`.
+
+Large-scale text is defined as:
+
+- Either **bold** text of at least 14 points (or 18.5 pixels).
+- Or regular text of at least 18 points (or 24 pixels).
+
+(see [WCAG definition of “large scale (text)”](https://www.w3.org/TR/WCAG21/#dfn-large-scale))
```

Comment:
This paragraph sort of "hangs in the air". I would write it like this:

> For more information, see [WCAG definition of "large scale (text)"](https://www.w3.org/TR/WCAG21/#dfn-large-scale).

Reply:
**To Do:** Rewrite.

Reply:
Done